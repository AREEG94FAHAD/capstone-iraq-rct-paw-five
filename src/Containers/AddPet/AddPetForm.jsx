import React from 'react'
import { useTranslation } from 'react-i18next'
import { Form, Col, Button } from 'react-bootstrap'
import PropTypes from 'prop-types'

function AddPetForm({
    handleSubmit,
    setFieldValue,
    file,
    handleChange,
    handleBlur,
    values,
    errors,
    touched,
}) {
    const { t } = useTranslation()
    const Values = values
    return (
        <div className="addpet-form">
            <h2 className="font-weight-bold text-center text-lg-left mb-5 mt-4">
                {t('add-pet.pets-info')}
            </h2>
            <Form onSubmit={handleSubmit}>
                <Form.Row>
                    <Col lg xl>
                        <Form.Control
                            onChange={handleChange}
                            name="petName"
                            onBlur={handleBlur}
                            value={values.petName}
                            placeholder={t('add-pet.pet-name')}
                        />
                        {touched.petName && errors.petName ? (
                            <div className="field-margin error-message">
                                {errors.petName}
                            </div>
                        ) : (
                            <div className="field-margin error-message"> </div>
                        )}
                    </Col>
                    <Col lg xl className="addpet-row-space">
                        <Form.Control
                            onChange={handleChange}
                            name="age"
                            onBlur={handleBlur}
                            value={values.age}
                            placeholder={t('add-pet.pet-age')}
                        />
                        <div className="field-margin"> </div>
                    </Col>
                </Form.Row>
                <Form.Row>
                    <Col lg xl>
                        <Form.Control
                            onChange={handleChange}
                            name="species"
                            onBlur={handleBlur}
                            value={values.species}
                            placeholder={t('add-pet.pet-species')}
                        />
                        {touched.species && errors.species ? (
                            <div className="field-margin error-message">
                                {errors.species}
                            </div>
                        ) : (
                            <div className="field-margin"> </div>
                        )}
                    </Col>
                    <Col lg xl className="addpet-row-space">
                        <Form.Control
                            onChange={handleChange}
                            name="weight"
                            onBlur={handleBlur}
                            value={values.weight}
                            placeholder={t('add-pet.pet-weight')}
                        />
                        <div className="field-margin"> </div>
                    </Col>
                </Form.Row>
                <Form.Row>
                    <Col lg xl>
                        <Form.Control
                            onChange={handleChange}
                            name="color"
                            onBlur={handleBlur}
                            value={values.color}
                            placeholder={t('add-pet.pet-color')}
                        />
                        {touched.color && errors.color ? (
                            <div className="field-margin error-message">
                                {errors.color}
                            </div>
                        ) : (
                            <div className="field-margin"> </div>
                        )}
                    </Col>
                    <Col lg xl className="addpet-row-space">
                        <p>{t('add-pet.gender')}</p>
                        <div className="d-flex justify-content-start">
                            <Form.Check
                                className="mr-1"
                                onChange={() => {
                                    Values.gender = 'male'
                                }}
                                type="radio"
                                onBlur={handleBlur}
                                name="gender"
                                value="Male"
                            />

                            <Form.Label>{t('add-pet.male')}</Form.Label>
                            <Form.Check
                                className="addpet-radio"
                                onChange={() => {
                                    Values.gender = 'female'
                                }}
                                type="radio"
                                name="gender"
                                value="Female"
                            />
                            <Form.Label>{t('add-pet.female')}</Form.Label>
                        </div>
                        <div className="field-margin"> </div>
                    </Col>
                </Form.Row>
                <Form.Row>
                    <Col lg xl>
                        <Form.Control
                            onChange={handleChange}
                            name="story"
                            onBlur={handleBlur}
                            value={values.story}
                            placeholder={t('add-pet.pet-story')}
                        />
                        {touched.story && errors.story ? (
                            <div className="field-margin error-message">
                                {errors.story}
                            </div>
                        ) : (
                            <div className="field-margin"> </div>
                        )}
                    </Col>
                    <Col lg xl className="addpet-row-space">
                        <div>
                            <Form.Label
                                htmlFor="file"
                                className=" text-center d-flex flex-column align-items-center justify-content-center bg-transparent custom-file-upload"
                            >
                                <p className="d-block text-center my-auto py-2">
                                    {t('add-pet.upload-photo')}
                                </p>
                            </Form.Label>
                        </div>
                        <input
                            id="file"
                            name="file"
                            type="file"
                            accept="images/*"
                            onChange={(e) => {
                                setFieldValue('file', e.currentTarget.files[0])
                            }}
                            className="d-none"
                        />

                        {file && (
                            <img
                                src={file}
                                className="upload-image"
                                alt="your pet"
                                width="200"
                                height="200"
                            />
                        )}
                    </Col>
                </Form.Row>
                <h2 className="font-weight-bold text-center text-lg-left mb-5 mt-5">
                    {t('add-pet.contact-info')}
                </h2>
                <Form.Row>
                    <Col lg xl>
                        <Form.Control
                            onChange={handleChange}
                            name="ownerName"
                            onBlur={handleBlur}
                            value={values.ownerName}
                            placeholder={t('add-pet.ownerName')}
                        />
                        {touched.ownerName && errors.ownerName ? (
                            <div className="field-margin error-message">
                                {errors.ownerName}
                            </div>
                        ) : (
                            <div className="field-margin"> </div>
                        )}
                    </Col>
                    <Col lg xl className="addpet-row-space">
                        <Form.Control
                            onChange={handleChange}
                            name="aboutOwner"
                            onBlur={handleBlur}
                            value={values.aboutOwner}
                            placeholder={t('add-pet.aboutOwner')}
                        />
                        <div className="field-margin"> </div>
                    </Col>
                </Form.Row>
                <Form.Row>
                    <Col lg xl>
                        <Form.Control
                            onChange={handleChange}
                            name="email"
                            type="email"
                            onBlur={handleBlur}
                            value={values.email}
                            placeholder={t('add-pet.email')}
                        />
                        {touched.email && errors.email ? (
                            <div className="field-margin error-message">
                                {errors.email}
                            </div>
                        ) : (
                            <div className="field-margin"> </div>
                        )}
                    </Col>
                    <Col lg xl className="addpet-row-space">
                        <Form.Control
                            onChange={handleChange}
                            name="phoneNumber"
                            onBlur={handleBlur}
                            value={values.phoneNumber}
                            placeholder={t('add-pet.phoneNumber')}
                        />
                        {touched.phoneNumber && errors.phoneNumber ? (
                            <div className="field-margin error-message">
                                {errors.phoneNumber}
                            </div>
                        ) : (
                            <div className="field-margin"> </div>
                        )}
                    </Col>
                </Form.Row>
                <Form.Row>
                    <Col lg xl>
                        <Form.Control
                            onChange={handleChange}
                            name="city"
                            onBlur={handleBlur}
                            value={values.city}
                            placeholder={t('add-pet.city')}
                        />
                        {touched.city && errors.city ? (
                            <div className="field-margin error-message">
                                {errors.city}
                            </div>
                        ) : (
                            <div className="field-margin"> </div>
                        )}
                    </Col>
                    <Col lg xl className="addpet-row-space">
                        <Form.Control
                            onChange={handleChange}
                            name="address"
                            onBlur={handleBlur}
                            value={values.address}
                            placeholder={t('add-pet.address')}
                        />
                        {touched.address && errors.address ? (
                            <div className="field-margin error-message">
                                {errors.address}
                            </div>
                        ) : (
                            <div className="field-margin"> </div>
                        )}
                    </Col>
                </Form.Row>
                <div className="d-flex d-flex justify-content-center">
                    <Button type="submit" className="btn">
                        {t('add-pet.submit')}
                    </Button>
                </div>
            </Form>
        </div>
    )
}
AddPetForm.propTypes = {
    handleSubmit: PropTypes.func.isRequired,
    setFieldValue: PropTypes.func.isRequired,
    handleChange: PropTypes.func.isRequired,
    handleBlur: PropTypes.func.isRequired,
    file: PropTypes.string.isRequired,
    values: PropTypes.arrayOf(PropTypes.object).isRequired,
    errors: PropTypes.arrayOf(PropTypes.object).isRequired,
    touched: PropTypes.arrayOf(PropTypes.object).isRequired,
}

export default AddPetForm
