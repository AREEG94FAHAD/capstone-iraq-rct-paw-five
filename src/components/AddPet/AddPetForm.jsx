import React from 'react'
import { useTranslation } from 'react-i18next'
import { Form, Col, Button } from 'react-bootstrap'
import PropTypes from 'prop-types'
import './AddPet.css'

function AddPetForm({
  handleSubmit,
  setFieldValue,
  file,
  handleChange,
  handleBlur,
  values,
  errors,
  touched,
}) {
  const { t } = useTranslation()
  const Values = values
  return (
    <div className="py-1 px-5" style={{ backgroundColor: '#f2f2f2' }}>
      <div className="px-lg-5">
        <h2
          className="font-weight-bold text-center text-lg-left mb-5 mt-4"
          style={{ color: '#e79559' }}
        >
          {t('add-pet.pets-info')}
        </h2>
        <Form onSubmit={handleSubmit}>
          <Form.Row>
            <Col lg xl>
              <Form.Control
                onChange={handleChange}
                name="petName"
                onBlur={handleBlur}
                value={values.petName}
                placeholder={t('add-pet.pet-name')}
                className={`${
                  touched.petName && errors.petName ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.petName && errors.petName ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.petName}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
            <Col lg xl className="a">
              <Form.Control
                onChange={handleChange}
                name="age"
                onBlur={handleBlur}
                value={values.age}
                placeholder={t('add-pet.pet-age')}
                className={`${
                  touched.age && errors.age ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.age && errors.age ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.age}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
          </Form.Row>
          <Form.Row>
            <Col lg xl>
              <Form.Control
                onChange={handleChange}
                name="species"
                onBlur={handleBlur}
                value={values.species}
                placeholder={t('add-pet.pet-species')}
                className={`${
                  touched.species && errors.species ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.species && errors.species ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.species}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
            <Col lg xl className="">
              <Form.Control
                onChange={handleChange}
                name="weight"
                onBlur={handleBlur}
                value={values.weight}
                placeholder={t('add-pet.pet-weight')}
                className={`${
                  touched.weight && errors.weight ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.weight && errors.weight ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.weight}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
          </Form.Row>
          <Form.Row>
            <Col lg xl>
              <Form.Control
                onChange={handleChange}
                name="color"
                onBlur={handleBlur}
                value={values.color}
                placeholder={t('add-pet.pet-color')}
                className={`${
                  touched.color && errors.color ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.color && errors.color ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.color}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
            <Col lg xl>
              <p>{t('add-pet.gender')}</p>
              <div className="d-flex justify-content-start mb-4">
                <Form.Check
                  className="mr-3"
                  onChange={() => {
                    Values.gender = 'male'
                  }}
                  type="radio"
                  onBlur={handleBlur}
                  name="gender"
                  value="Male"
                />

                <Form.Label className="mr-3">{t('add-pet.male')}</Form.Label>
                <Form.Check
                  className="mr-3"
                  onChange={() => {
                    Values.gender = 'female'
                  }}
                  type="radio"
                  name="gender"
                  value="Female"
                />
                <Form.Label className="mr-3">{t('add-pet.female')}</Form.Label>
              </div>
              <div className="mb-5"> </div>
            </Col>
          </Form.Row>
          <Form.Row>
            <Col lg xl>
              <Form.Control
                onChange={handleChange}
                name="story"
                onBlur={handleBlur}
                value={values.story}
                placeholder={t('add-pet.pet-story')}
                className={`${
                  touched.story && errors.story ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.story && errors.story ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.story}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
            <Col lg xl className="">
              <div>
                <Form.Label
                  htmlFor="file"
                  className=" text-center d-flex flex-column align-items-center justify-content-center bg-transparent border rounded uplod-border rounded-3 w-75 w-form mt-3"
                >
                  <p className="d-block text-center my-auto py-2">
                    {t('add-pet.upload-photo')}
                  </p>
                </Form.Label>
              </div>
              <input
                id="file"
                name="file"
                type="file"
                accept="images/*"
                onChange={e => {
                  setFieldValue('file', e.currentTarget.files[0])
                }}
                className="d-none"
              />

              {file && (
                <img
                  src={file}
                  className="d-block mt-4 mx-auto"
                  alt="your pet"
                  width="200"
                  height="200"
                />
              )}
            </Col>
          </Form.Row>
          <h2
            className="font-weight-bold text-center text-lg-left mb-5 mt-5"
            style={{ color: '#e79559' }}
          >
            {t('add-pet.contact-info')}
          </h2>
          <Form.Row>
            <Col lg xl>
              <Form.Control
                onChange={handleChange}
                name="ownerName"
                onBlur={handleBlur}
                value={values.ownerName}
                placeholder={t('add-pet.ownerName')}
                className={`${
                  touched.ownerName && errors.ownerName ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.ownerName && errors.ownerName ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.ownerName}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
            <Col lg xl className="addpet-row-space">
              <Form.Control
                onChange={handleChange}
                name="aboutOwner"
                onBlur={handleBlur}
                value={values.aboutOwner}
                placeholder={t('add-pet.aboutOwner')}
                className=" mb-5 form-borders rounded-0 border-0 w-75 w-form"
              />
            </Col>
          </Form.Row>
          <Form.Row>
            <Col lg xl>
              <Form.Control
                onChange={handleChange}
                name="email"
                type="email"
                onBlur={handleBlur}
                value={values.email}
                placeholder={t('add-pet.email')}
                className={`${
                  touched.email && errors.email ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.email && errors.email ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.email}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
            <Col lg xl>
              <Form.Control
                onChange={handleChange}
                name="phoneNumber"
                onBlur={handleBlur}
                value={values.phoneNumber}
                placeholder={t('add-pet.phoneNumber')}
                className={`${
                  touched.phoneNumber && errors.phoneNumber ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.phoneNumber && errors.phoneNumber ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.phoneNumber}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
          </Form.Row>
          <Form.Row>
            <Col lg xl>
              <Form.Control
                onChange={handleChange}
                name="city"
                onBlur={handleBlur}
                value={values.city}
                placeholder={t('add-pet.city')}
                className={`${
                  touched.city && errors.city ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.city && errors.city ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.city}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
            <Col lg xl className="addpet-row-space">
              <Form.Control
                onChange={handleChange}
                name="address"
                onBlur={handleBlur}
                value={values.address}
                placeholder={t('add-pet.address')}
                className={`${
                  touched.address && errors.address ? 'mb-2' : 'mb-5'
                } form-borders rounded-0 border-0 w-75 w-form`}
              />
              {touched.address && errors.address ? (
                <div className="mb-2 text-center text-lg-left text-danger">
                  {errors.address}
                </div>
              ) : (
                <div className="mb-5"> </div>
              )}
            </Col>
          </Form.Row>
          <div className="d-flex d-flex justify-content-center mr-lg-5">
            <Button
              type="submit"
              className="btn px-5 mr-lg-5 my-4 text-dark border-0"
              style={{ backgroundColor: '#e79559' }}
            >
              {t('add-pet.submit')}
            </Button>
          </div>
        </Form>
      </div>
    </div>
  )
}
AddPetForm.propTypes = {
  handleSubmit: PropTypes.func.isRequired,
  setFieldValue: PropTypes.func.isRequired,
  handleChange: PropTypes.func.isRequired,
  handleBlur: PropTypes.func.isRequired,
  file: PropTypes.string.isRequired,
  values: PropTypes.arrayOf(PropTypes.object).isRequired,
  errors: PropTypes.arrayOf(PropTypes.object).isRequired,
  touched: PropTypes.arrayOf(PropTypes.object).isRequired,
}

export default AddPetForm
