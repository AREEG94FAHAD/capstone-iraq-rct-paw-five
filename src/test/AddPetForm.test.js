import React from 'react'
import renderer from 'react-test-renderer'
import AddPetForm from './../../components/AddPet/AddPetForm'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <AddPetForm
        handleSubmit
        setFieldValue
        file
        handleChange
        handleBlur
        values
        errors
        touched
      />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
