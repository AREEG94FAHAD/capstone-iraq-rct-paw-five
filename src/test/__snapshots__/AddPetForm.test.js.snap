// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="overflow-hidden"
>
  <div
    className="mb-4 row"
  >
    <div
      className="pt-5 pl-lg-5 ml-lg-4 col"
    >
      <div
        className="d-flex flex-column-reverse align-items-center flex-md-row justify-content-md-center justify-content-lg-start mt-4 mb-4"
      >
        <img
          alt="paw"
          className="mt-4 mt-md-0 mr-lg-2 "
          height="55"
          src="paw.svg"
          width="55"
        />
        <h1
          className="font-weight-bold fs-1 text-center text-md-left px-2"
          style={
            Object {
              "color": "#e79559",
            }
          }
        >
          add-pet.add for adoption
        </h1>
      </div>
      <div
        className="text-muted w-75 w-form ml-lg-5 pl-lg-2 mb-4"
      >
        <h4
          className="px-3 text-center text-lg-left"
        >
          add-pet.make-pets-happy
        </h4>
      </div>
    </div>
    <div
      className="d-flex justify-content-end ml-lg-5 col-lg-4"
    >
      <img
        alt="cat"
        className="w-100 cat-image"
        height="450"
        src="AddPetCat.svg"
      />
    </div>
  </div>
  <div
    className="py-1 px-5"
    style={
      Object {
        "backgroundColor": "#f2f2f2",
      }
    }
  >
    <div
      className="px-lg-5"
    >
      <h2
        className="font-weight-bold text-center text-lg-left mb-5 mt-4"
        style={
          Object {
            "color": "#e79559",
          }
        }
      >
        add-pet.pets-info
      </h2>
      <form
        className=""
        onSubmit={[Function]}
      >
        <div
          className="form-row"
        >
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="petName"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.pet-name"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
          <div
            className="a col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="age"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.pet-age"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
        </div>
        <div
          className="form-row"
        >
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="species"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.pet-species"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="weight"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.pet-weight"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
        </div>
        <div
          className="form-row"
        >
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="color"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.pet-color"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
          <div
            className="col-xl col-lg"
          >
            <p>
              add-pet.gender
            </p>
            <div
              className="d-flex justify-content-start mb-4"
            >
              <div
                className="mr-3 form-check"
              >
                <input
                  className="form-check-input position-static"
                  disabled={false}
                  name="gender"
                  onBlur={[Function]}
                  onChange={[Function]}
                  type="radio"
                  value="Male"
                />
              </div>
              <label
                className="mr-3 form-label"
              >
                add-pet.male
              </label>
              <div
                className="mr-3 form-check"
              >
                <input
                  className="form-check-input position-static"
                  disabled={false}
                  name="gender"
                  onChange={[Function]}
                  type="radio"
                  value="Female"
                />
              </div>
              <label
                className="mr-3 form-label"
              >
                add-pet.female
              </label>
            </div>
            <div
              className="mb-5"
            >
               
            </div>
          </div>
        </div>
        <div
          className="form-row"
        >
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="story"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.pet-story"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
          <div
            className="col-xl col-lg"
          >
            <div>
              <label
                className=" text-center d-flex flex-column align-items-center justify-content-center bg-transparent border rounded uplod-border rounded-3 w-75 w-form mt-3 form-label"
                htmlFor="file"
              >
                <p
                  className="d-block text-center my-auto py-2"
                >
                  add-pet.upload-photo
                </p>
              </label>
            </div>
            <input
              accept="images/*"
              className="d-none"
              id="file"
              name="file"
              onChange={[Function]}
              type="file"
            />
          </div>
        </div>
        <h2
          className="font-weight-bold text-center text-lg-left mb-5 mt-5"
          style={
            Object {
              "color": "#e79559",
            }
          }
        >
          add-pet.contact-info
        </h2>
        <div
          className="form-row"
        >
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="ownerName"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.ownerName"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
          <div
            className="addpet-row-space col-xl col-lg"
          >
            <input
              className=" mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="aboutOwner"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.aboutOwner"
              value=""
            />
          </div>
        </div>
        <div
          className="form-row"
        >
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.email"
              type="email"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="phoneNumber"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.phoneNumber"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
        </div>
        <div
          className="form-row"
        >
          <div
            className="col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="city"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.city"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
          <div
            className="addpet-row-space col-xl col-lg"
          >
            <input
              className="mb-5 form-borders rounded-0 border-0 w-75 w-form form-control"
              name="address"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="add-pet.address"
              value=""
            />
            <div
              className="mb-5"
            >
               
            </div>
          </div>
        </div>
        <div
          className="d-flex d-flex justify-content-center mr-lg-5"
        >
          <button
            className="btn px-5 mr-lg-5 my-4 text-dark border-0 btn btn-primary"
            disabled={false}
            style={
              Object {
                "backgroundColor": "#e79559",
              }
            }
            type="submit"
          >
            add-pet.submit
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
